index.js

```diff
@@ -52,7 +52,7 @@ MS.metadata({
   dispatchDocsLatest: "1.4",
   kommanderDocsLatest: "2.2",
   konvoyDocsLatest: "2.2",
-  kaptainDocsLatest: "1.3.0",
+  kaptainDocsLatest: "2.2.0",
```

Comment:
```suggestion
  kaptainDocsLatest: "2.0.0",
```
was this 2.2 all the time? lol..